### DISCUSSION TITLE
Tire Rotating / Set tracking

### DISCUSSION THREAD

**Discussion by DiscoSlug** [hargata/lubelog#308] [Ideas]
Hi, 

This idea is because of my tracking tire sets on multiple vehicles, including 3 sets for a Jeep. 

It would be great if we could create a service record that puts a set of tires into use (winter vs. summer for example) and we enter the odometer reading. Later in the year when we switch and enter the odometer reading, that logs the number of miles in between the switch to the swapped out tires and starts the odometer count for the new set. I see this method as allowing me to know approximately how many miles I'm getting out of a tire and when I should be expecting to replace it. 

An another alternative that is likely the more responsible owner method, is to instead have a tire specific tab that can track multiple sets of tires with tread depth tracking and if I want, tracking a date/odometer reading when I add the tires or remove them. This allows me to make the only truly quantitative measurement for deciding when to replace a tire or set.

**Comment by waif69** [comment]
I see that I'm not the only one who would like to track that information. I support this request.

**Comment by hargata** [comment]
@DiscoSlug @waif69 please see #382 for an update on this.

**Comment by iamdabe** [comment]
Came here to share the same thought - it would be great to have a dedicated feature for tire rotation and managing winter/summer tires. For now, I'll go ahead and implement the solution mentioned in #382.

It would be fantastic to see a more user-friendly approach in the future, or even something more generic—a feature to track items installed on the car that aren't in use all the time. This could include a list of items (similar to supplies), like trailers, wheels, or other accessories, that incur wear and tear by odometer usage. Having these tracked in a dedicated space would be incredibly useful.

Thanks for providing a workaround for now

**Reply by DiscoSlug** [reply]
I personally like the quantitative aspect of specification measurements for things like brake pad thickness and tire tread depth so we can keep tabs on the wear.

**Reply by chirmstream** [reply]
I agree I'd like a little more control & info for tracking tire and brake pad wear.  I'll use the solution shown in #382 but I think it will get clunky.  My usage for tires is likely different from most, since I keep track of costs on my race car and may be purchasing one or two sets of the same tire per year (along side another winter/offseason set).

If you look at the Road Trip app and their tire logging, this more in-line with how I'd imagine the tire management to work.
https://darrensoft.ca/roadtrip/manual/tires/
I've been enjoying LubeLogger a lot though, and appreciate all the work that has been put into it!

**Comment by hargata** [comment]
Please see https://docs.lubelogger.com/Records/Equipment/ where you can attach Equipment to Odometer records and track total distance traveled on each equipment or in this case, each set of tires.
